.github/workflows/release.yml

```
@@ -27,38 +94,34 @@ jobs:
 
       - name: Checkout the repo
         uses: actions/checkout@master
-
-      - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          path: cli
 
-      # TODO: determine if it is possible to break out the build into multiple targets, to enable parallel job execution
-      - name: Build Distributions
-        run: cargo xtask dist
+      - name: Download release artifacts
+        uses: actions/download-artifact@v3
 
-      - name: Sign binaries
+      - name: Sign binaries and prep archives
         run: |
           echo -e $MINISIGN_KEY > minisign.key
-          echo $MINISIGN_PASSWORD | minisign -Sm target/dist/phylum-linux-x86_64/phylum -s minisign.key -t 'Phylum - the future of software supply chain security'
-          echo $MINISIGN_PASSWORD | minisign -Sm target/dist/phylum-macos-x86_64/phylum -s minisign.key -t 'Phylum - the future of software supply chain security'
-          echo $MINISIGN_PASSWORD | minisign -Sm target/dist/phylum-macos-aarch64/phylum -s minisign.key -t 'Phylum - the future of software supply chain security'
+          for archive in phylum-*/;
+          do
+            archive=$(echo ${archive} | sed -e "s/\/$//")
+            echo $MINISIGN_PASSWORD | minisign -Sm ${archive}/phylum -s minisign.key -t 'Phylum - the future of software supply chain security'
+            cp -R shell-completions ${archive}/completions
+            cp cli/cli/src/install.sh ${archive}/install.sh
+            cp cli/cli/src/settings.yaml ${archive}/settings.yaml
```

```suggestion
            cp cli/cli/src/settings.yaml ${archive}/settings.yaml
            chmod 600 ${archive}/settings.yaml
```
This would be a good spot to restrict the file permissions of the `settings.yaml` file. It contains an `offline_access` token that should only be visible to the user.

---

Hmmm... I wasn't aware that was a problem. The CLI also now creates the `settings.yaml` file if it doesn't already exist. I think this is worth an issue of its own.

---

Create #208 to look into this.